cloudbuild.yaml

```diff
@@ -91,16 +91,16 @@ steps:
         -var=project_id=$PROJECT_ID \
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=cluster_location=$_REGION \
-        -var=kubernetes_namespace=ml-$SHORT_SHA-$_BUILD_ID \
+        -var=kubernetes_namespace=ml-$SHORT_SHA-$_BUILD_ID-ray \
```

Comment:
Changing the namespace allows the steps to run independant from each other, but do we need additional changes to run the steps in parallel? I couldn't tell from cloudbuild logs if the steps are starting in parallel

Reply:
Oh I see, I guess just changing the `waitFor` in the test steps makes it parallel

Reply:
Yes, changing the `waitFor` explicitly will tell it to run right after the steps within `waitFor`